packages/superset-ui-number-format/test/factories/createSiAtMostNDigitFormatter.test.ts

```diff
@@ -30,17 +30,17 @@ describe('createSiAtMostNDigitFormatter({ n })', () => {
     const formatter = createSiAtMostNDigitFormatter();
     expect(formatter(10)).toBe('10');
     expect(formatter(1)).toBe('1');
-    expect(formatter(1.0)).toBe('1');
-    expect(formatter(10.0)).toBe('10');
+    expect(formatter(1)).toBe('1');
+    expect(formatter(10)).toBe('10');
```

Comment:
ditto prettier removed the `.0`s  in this file

Reply:
added `prettier-ignore`